rest/route/task_generate.go

```diff
@@ -38,6 +40,10 @@ func (h *generateHandler) Parse(ctx context.Context, r *http.Request) error {
 	}
 	h.taskID = gimlet.GetVars(r)["task_id"]
 
+	if err = validateFileSize(h.files, h.env.Settings().TaskLimits.MaxGenerateTaskJSONSize); err != nil {
```

Comment:
you can just return the wrapped error here instead of returning nil since they're equivalent

Reply:
done